apps/studio/src/features/editing-experience/components/CreateFolderModal/CreateFolderModal.tsx

```diff
@@ -45,7 +44,26 @@ export const CreateFolderModal = ({
   siteId,
   parentFolderId,
 }: CreateFolderModalProps): JSX.Element => {
-  const { setValue, register, handleSubmit, watch, formState, getFieldState } =
+  return (
+    <Modal isOpen={isOpen} onClose={onClose}>
+      <ModalOverlay />
+      <CreateFolderModalContent
+        key={String(isOpen)}
+        isOpen={isOpen}
```

Comment:
this is no longer needed right (unsued props)?
maybe actually instead of key, just use `isOpen && <CreateFolderModalContent` ?

Reply:
Same for the rest?